mlflow/metrics/genai/genai_metric.py

```diff
@@ -295,14 +295,18 @@ def score_model_on_one_payload(
             try:
                 raw_result = model_utils.score_model_on_payload(eval_model, payload)
                 return _extract_score_and_justification(raw_result)
+            except ImportError as e:
+                raise e
```

Comment:
```suggestion
            except ImportError:
                raise
```

nit: e can be removed